api/Dockerfile

```diff
@@ -90,6 +90,15 @@ COPY api.env api.env
 COPY postgres.env postgres.env
 COPY phpstan.neon .
 
+# Behat
+RUN mkdir -p /tmp/html
+RUN mkdir -p /tmp/sql
+
+RUN ["chown", "www-data", "/var/www/tests/Behat/run-smoke-tests.sh"]
```

Comment:
Nothing wrong with doing it this way but maybe a bit inconsistent doing the array type commands here but just standard spaces elsewhere? :)

Reply:
Yeah good point - it was a copy-paste from the other Dockerfile. I shall update 👍